src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -21,7 +20,7 @@ export default function WebAuthnScreen(props) {
 	return (
 		<Container>
 			<Row className="justify-content-center">
-				<Col className="col-webauthn-card">
+				<Col lg={11} className="col-webauthn-card">
```

Comment:
@Aringoaway Can this be the size of the column set in the scss file?

Reply:
@Pe5h4 why this column size should be set through the scss styles? We use this format in all components, and it's also set up for media styles